ansible_base/authentication/serializers/authenticator.py

```diff
@@ -146,3 +146,12 @@ def _get_related(self, obj) -> dict[str, str]:
             related['users'] = get_relative_url('authenticator-users-list', kwargs={'pk': obj.pk})
 
         return related
+
+
+class AuthenticatorUpdateSerializer(AuthenticatorSerializer):
+    """Specialized serializer for update operations that makes type field read-only."""
+
+    type = ChoiceField(choices=get_authenticator_plugins(), read_only=True, help_text="Cannot be changed after creation.")
```

Comment:
Won't this _replace_ the existing help text in the schema delivered for OpenAPI?

https://github.com/ansible/django-ansible-base/blob/43982875957bf65ab9df99798387892d049a9450/ansible_base/authentication/models/authenticator.py#L24-L28

Sure it's important to know that the field can't be changed after creation, but formally this is documented by the surrounding metadata / schema, and the application function of the field would seem to be more important to me.